transforms/language/html2parquet/dpk_html2parquet/transform.py

```diff
@@ -236,3 +236,35 @@ def apply_input_params(self, args: Namespace) -> bool:
         self.params = self.params | captured
         logger.info(f"html2parquet parameters are : {self.params}")
         return True
+
+
+class Html2Parquet(Html2ParquetTransform):
```

Comment:
This should also inherit from AbstractTableTransform to indicate that it implements the transform() method below.

Reply:
@daw3rd I admit, This is one that I struggled a lot with. The conclusion I came to: Only methods that are invoked from the runtime need to implement the abstraction.  For now, this method is not intended to be called from the runtime so I don't see the benefit making derived from the AbstractTableTransform.